guides/client/node.mdx

```diff
@@ -145,7 +145,18 @@ There are two types of nodes in Quai: **Global** and **Slice** nodes.
 
 ### Environment Variables
 
-After you've successfully installed go-quai, you'll need to configure your node. Go-quai uses a `config.toml` file to configure the node. The `config.toml` file is not located in the `go-quai` but rather in a directory at the root of your machine.
+After you've successfully installed go-quai, you'll need to configure your node. Go-quai uses a `config.toml` file to configure the node. If this is your first time running go-quai, you'll need to run the following command to populate the config file:
+
+```bash
+./build/bin/go-quai config
+```
+
+<Warning>
+	You only need to run this command once the first time you run go-quai. Once the config file has been created, you do not need to run the
+	configuration command again.
+</Warning>
```

Comment:
There's no risk or harm to running it multiple times. I would change this from warning to info so people don't think it resets, and try to run this command to clear their config.

Reply:
Gotcha, this makes sense.